applications/MeshingApplication/tests/cpp_tests/mmg/test_internal_interpolation_process.cpp

```diff
@@ -173,6 +174,7 @@ namespace Kratos
             Properties::Pointer p_elem_prop = r_model_part.CreateNewProperties(0);
             // In case the StructuralMechanicsApplciation is not compiled we skip the test
             KRATOS_SKIP_TEST_IF_NOT(KratosComponents<Element>::Has("UpdatedLagrangianElement2D3N")) << "This test needs the StructuralMechanicsApplication" << std::endl;
+            KRATOS_SKIP_TEST_IF_NOT(KratosComponents<ConstitutiveLaw>::Has("SmallStrainJ2PlasticityPlaneStrain2DLaw")) << "This test needs the ConstitutiveLawsApplication" << std::endl;
```

Comment:
@loumalouomega cannot this be done with a simple elastic law?

Reply:
![imagen](https://user-images.githubusercontent.com/19991680/109208172-4c1c4980-77aa-11eb-8a52-f3313507b320.png)
No, it requires some sort of historical record (aka plasticity)

Reply:
Ok